examples/quickstart-mlcube/dev/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:22.04
+LABEL author="Marius Schlegel"
```

Comment:
We usually just put "The Flower Authors" to indicate that this is a community effort

Reply:
Thank you, @danieljanes, I have changed it accordingly.

Reply:
Thanks for the fast update!